pyproject.toml

```diff
@@ -63,6 +63,7 @@ dependencies = [
   "toolz",
   "xarray >=2023.11.0, !=2024.6.0, <2024.10.0", # FIXME: 2024.10.0 breaks rechunker with zarr
   "xclim >=0.53.2, <0.55",
+  "xsdba@git+https://git@github.com/Ouranosinc/xsdba.git@naming_conventions",
```

Comment:
FYI, this will not pass the `PyPI` smell test. Pointing to git branches for dependencies is forbidden on `PyPI` (thankfully).

We can release a patch when the branch is merged.

Reply:
Yep. Exactly that.